doc/source/deployment/setup-deployer.rst

```diff
@@ -0,0 +1,179 @@
+.. _setupdeployer:
+
+Set Up Deployer
+=================
+
+.. blockdiag::
+
+   blockdiag {
+     default_fontsize = 11;
+     deployer [label="Setup deployer"]
+     ses_integration [label="SES Integration"]
+     configure [label="Configure\nCloud"]
+     setup_caasp_workers [label="Setup CaaS Platform\nworker nodes"]
+     patch_upstream [label="Apply patches\nfrom upstream\n(for developers)"]
+     build_images [label="Build Docker images\n(for developers)"]
+     deploy_airship [label="Deploy Airship"]
+     deploy_openstack [label="Deploy OpenStack"]
+
+     group {
+       deployer
+       color="red"
+     }
+
+     deployer -> ses_integration;
+     ses_integration -> configure;
+     configure -> setup_caasp_workers;
+
+     group {
+       color = "#EEEEEE"
+       label = "Cloud Deployment"
+       setup_caasp_workers -> patch_upstream;
+       patch_upstream -> build_images;
+       build_images -> deploy_airship [folded];
+       setup_caasp_workers -> deploy_airship;
+       deploy_airship -> deploy_openstack;
+     }
+   }
+
+
+Base Software
+-------------
+
+The following software must be installed on your `Deployer`:
```

Comment:
If they install it via rpm package, this requirements should be brought automatically as package dependencies

Reply:
recommend addressing after merge